docassemble/MLHPPOAndProposedOrder/data/questions/ppo_and_proposed_order.yml

```diff
@@ -1169,26 +1169,6 @@ fields:
   - code: |
       petitioner_other_children[i].name_fields()
 ---
-id: petitioner_other_children birthdate
```

Comment:
Note, this block was deleted because it duplicates another nearly identical block.